docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All objects of type "user" are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+Being explicit about the wildcard type restrictions also lets you model scenarios like “all employees can see this document, but not all external users”, “all user accounts can access this document, but not service/machine-to-machine accounts”.
+
+This change implies that you’ll need to change your code to write tuples with this new syntax, and that you’ll need to migrate existing tuples to use the new format.
+
+You might have 3 kinds of tuples in your model that use “\*”, with different migration strategies:
+
+1. Tuples that have user = “\*”
+
+You would need to retrieve those tuples and write them using the proper type (e.g. `user:*`). To retrieve them, you’ll need to use the Read endpoint, filter on your side the tuples that have `user = “*”`, and call the Write API for each one, with the proper type, e.g:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All objects of type "user" are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+2. Tuples that have `user = "employee:*”`, where `employee` is NOT a type that is defined in the new iteration of your model.
+
+If you have tuples with this format, they will be considered invalid because they don’t have a corresponding type in the model. If you need such a type defined, you’ll need to add it to the model, and the scenario will be similar to the one described below.
+
+3. Tuples that have `user = “user:*”`, which would mean "the user with user_id = '\*'”, where `user` is type that is defined in the new iteration of your model.
+
+In this case, the meaning of the tuple will change. If you were intending to specify "a user with user id = \*", you will need to encode it in a different way instead of using “\*”. If you intended to specify “every user has this relationship with this object” then it’s not the way it would have worked with schema version = 1.0, but it will work with version = 1.1.
+
+:::caution Warning
+If you have any wildcard tuples (i.e., `*` or `user:*`) that were created with model schema 1.0, you <strong><em>must</em></strong> delete and re-add these tuples with the appropriate type. This allows <ProductName format={ProductNameFormat.ShortForm} /> to interpret these tuples appropriately with the model schema 1.1 semantics. Failure to delete and re-add may cause <ProductName format={ProductNameFormat.ShortForm} /> to interpret these tuples incorrectly.
+:::
+
+## Query Evaluation Behavior with Type Restrictions
+
+When you make changes to a model that already has tuples, those tuples might become invalid. Some cases where this can happen are:
```

Comment:
You can't change a model and model's don't have tuples. I think what we're trying to say here is

"It is possible to introduce new models and have existing tuples (from prior models) with incompatibilities with the new model. Some cases where this can happen include:"

...